src/abstractdataframe/show.jl

```diff
@@ -597,50 +597,64 @@ function _show(io::IO,
                rowlabel::Symbol = :Row,
                summary::Bool = true,
                eltypes::Bool = true,
-               rowid=nothing,
-               truncstring::Int)
+               truncstring::Int = 32,
```

Comment:
You have left `splitcols` in https://github.com/JuliaData/DataFrames.jl/pull/2429/files#diff-38a6c4f0cb7743ccc6d4e8470ebdbd11L596. We should just remove it (@nalimilan I know it is "hard breakage", but I think there is no better way to handle it cleanly).

Reply:
I think I removed all of them. I also tested and all the types seems to be working fine (GroupedDataFrames, DataFrameRow, etc.). However, I need help to test.